services/src/test/java/io/scalecube/services/ServiceRemoteTest.java

```diff
@@ -283,7 +285,7 @@ public void test_remote_bidi_greeting_expect_IllegalArgumentException() {
     // call the service.
     StepVerifier.create(responses)
         .expectErrorMessage("IllegalArgumentException")
-        .verify(Duration.ofSeconds(3));
+        .verify(Duration.ofSeconds(300));
```

Comment:
is it really needed?

Reply:
its not good idea to change this!
its a bug in reactor and it needs to be fixed there first
so lets just wait until its fixed.

Reply:
https://github.com/rsocket/rsocket-java/pull/579